src/renderer/features/fellowship-voting-history/model/votes.ts

```diff
@@ -12,20 +11,23 @@ import { votingHistoryFeatureStatus } from './status';
 
 const gate = createGate<{ referendumId: ReferendumId | null }>({ defaultState: { referendumId: null } });
 
-const $votes = fellowshipModel.$store.map(store => store?.votes ?? []);
-
-const $votesList = combine($votes, gate.state, (votes, { referendumId }) => {
-  if (!votes || referendumId === null) return [];
+const $votesMap = fellowshipModel.$store.map(store => store?.votes ?? []);
```

Comment:
It's still an object, not array. Our typescript settings are not strict enough for caching this type of bugs